app/src/main/java/com/sschuraytz/charcoaldrawing/MainActivity.java

```diff
@@ -32,11 +34,13 @@ protected void onCreate(Bundle savedInstanceState) {
         setUpRedo();
     }
 
+    @SuppressLint("ClickableViewAccessibility")
```

Comment:
I've seen mixed reviews about this online. Some say it's better to override performClick(). I opted for this solution, but I would like to hear your opinion.